src/lib/infusion/src/components/inlineEdit/css/InlineEdit.css

```diff
@@ -0,0 +1,64 @@
+/* Themed Appearance + Themed States */
```

Comment:
I'll see what I can do about this. I foresee some complexity here in regards to how Infusion is currently distributed on NPM vs. other packages:

- installing Infusion from NPM does not result in a `/dist` directory or similar that contains concatenated files; I will need to make a custom build myself
- that will require entering `node_modules/infusion`, running `npm install` again to get the grunt dependencies, then running the build task
- the Infusion build task currently produce a zip file, without the option for other kinds of output that would allow for straightforward copying of the relevant files

None of these are insurmountable, but add complexity; I'll work through them and may make a PR against the current Infusion build tasks if appropriate.